docs/quickstart.rst

```diff
@@ -259,13 +259,13 @@ This section describes how to deploy the REST Proxy container and then consume d
 
   .. sourcecode:: bash
 
-    docker run -it --net=host --rm confluentinc/cp-schema-registry:3.3.0-SNAPSHOT bash
+    $ docker run -it --net=host --rm confluentinc/cp-schema-registry:3.3.0-SNAPSHOT bash
 
   The first step in consuming data via the REST Proxy is to create a consumer instance.
 
   .. sourcecode:: bash
 
-    curl -X POST -H "Content-Type: application/vnd.kafka.v1+json" \
+    # curl -X POST -H "Content-Type: application/vnd.kafka.v1+json" \
```

Comment:
Same explanation. Anything that runs after we've run `docker run ... bash` will give the user root shell.